addons/website/static/src/less/website.snippets.less

```diff
@@ -11,6 +11,15 @@
     }
 }
 
+// Inner content Snipptes
+#wrapwrap
+    .row
+        [class^="col-md-"]
+            .s_auto_height
+            {
+                height: auto;
+            }
```

Comment:
Nope in theme_common, height is 100% for many snippets. So this class is added to the inner content snippets (to prevent issues like : https://drive.google.com/file/d/0B1uIL9E_zXrrNmZSZGZ5dDZvdFU/view )